packages/web-components/src/components/va-search/va-search.tsx

```diff
@@ -79,6 +84,61 @@ export class VaSearch {
   })
   buttonKeyDownEvent: EventEmitter;
 
+  /**
+   * Fires when a suggestion is clicked
+   */
+  @Event({
+    composed: true,
+    bubbles: true,
+  })
+  suggestionClickEvent: EventEmitter;
+
+  /**
+   * Fires when a suggestion is focused
+   */
+  @Event({
+    composed: true,
+    bubbles: true,
+  })
+  suggestionFocusEvent: EventEmitter;
+
+  /**
+   * Fires when a suggestion is focused and a key is pressed
+   */
+  @Event({
+    composed: true,
+    bubbles: true,
+  })
+  suggestionKeyDownEvent: EventEmitter;
+
+  /**
+   * Fires when a suggestion is pressed
+   */
+  @Event({
+    composed: true,
+    bubbles: true,
+  })
+  suggestionMouseDownEvent: EventEmitter;
+
+  /**
+   * Fires when user moves cursor over the suggestion
+   */
+  @Event({
+    composed: true,
+    bubbles: true,
+  })
+  suggestionMouseOverEvent: EventEmitter;
+
+  /**
+   * An array of formatted suggestions
+   */
+  @State() formattedSuggestions: string[] = [];
+
+  /**
+   * Index of selected suggestion
+   */
+  @Prop() activeIndex?: number;
```

Comment:
Are we expecting engineers to be setting `activeIndex` when they use the component?